src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int weight;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. The field 'weight' should be protected or private to prevent direct access to the field. Consider using private and providing a getter and setter for it.